futaba/converters/annotations.py

```diff
@@ -0,0 +1,57 @@
+#
+# annotations.py
+#
+# futaba - A Discord Mod bot for the Programming server
+# Copyright (c) 2017-2018 Jake Richardson, Ammon Smith, jackylam5
+#
+# futaba is available free of charge under the terms of the MIT
+# License. You are free to redistribute and/or modify it under those
+# terms. It is distributed in the hopes that it will be useful, but
+# WITHOUT ANY WARRANTY. See the LICENSE file for more details.
+#
+
+'''
+Module for managing the command annotations
+'''
+
+__all__ = [
+    'ANNOTATIONS',
+]
+
+ANNOTATIONS = {
+    # Built-in
+    'int': 'Integer',
+    'str': 'String',
+    'bool': 'Boolean',
+
+    # Discord.py
+    'ClientUser': 'Bot Client',
+    'RelationShip': 'User Relationship',
```

Comment:
Is it actually `RelationShip` or is this a typo?

Reply:
![image](https://user-images.githubusercontent.com/18365785/46260862-ab93a380-c4e3-11e8-99c7-0f60116aa8a6.png)

Reply:
Ahhh, Yes it is